src/renderer/views/multiplayer/custom.vue

```diff
@@ -159,15 +175,101 @@ const battles = computed(() => {
         return true;
     });
 
+    const scoredBattles = battles.map(scoreBattle);
+
     if (selectedBattle.value === null) {
-        const biggestBattle = battles.sort((a, b) => b.playerCount.value - a.playerCount.value)[0];
         // eslint-disable-next-line vue/no-side-effects-in-computed-properties
-        selectedBattle.value = biggestBattle;
+        selectedBattle.value = scoredBattles[0];
     }
 
-    return battles;
+    return scoredBattles;
 });
 
+function scoreBattle(battle: SpadsBattle) {
+    let score = 0;
+    let factors = {};
```

Comment:
Could you try typing this object? I believe this will probably fail the type-check anyway

Reply:
Oh it didn't fail the type check 🤔 Either way more explicit typing would be appreciated here